pkg/controller/mpi_job_controller.go

```diff
@@ -364,6 +376,13 @@ func NewMPIJobControllerWithClock(
 		UpdateFunc: controller.handleObjectUpdate,
 		DeleteFunc: controller.handleObject,
 	})
+	if priorityClassInformer != nil {
+		priorityClassInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+			AddFunc:    controller.handleObject,
+			UpdateFunc: controller.handleObjectUpdate,
+			DeleteFunc: controller.handleObject,
```

Comment:
Does it make sense to add an event handler for priorityClass? They are not owned by the MPIJob.

Reply:
You're right.

Reply:
Done.